src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala

```diff
@@ -268,7 +268,11 @@ class Silver(_workspace: Workspace, _database: Database, _config: Config)
       )
   }
 
-  lazy private[overwatch] val jobRunsModule = Module(2011, "Silver_JobsRuns", this, Array(1004, 2010, 2014), shuffleFactor = 12.0)
+  lazy private[overwatch] val jobRunsModule =
+    Module( 2011, "Silver_JobsRuns", this, Array(1004, 2010, 2014), shuffleFactor = 12.0)
+      .withSparkOverrides( Map(
+        "spark.databricks.adaptive.autoOptimizeShuffle.enabled" -> "true"))
+
```

Comment:
Can we  create a silverJobRunsSparkOverrides variable and put "spark.databricks.adaptive.autoOptimizeShuffle.enabled" -> "true" inside the variable and use this in .withSparkOverrides.
ex:
  private val sparkExecutorsSparkOverrides = Map(
    "spark.databricks.delta.optimizeWrite.numShuffleBlocks" -> "500000",
    "spark.databricks.delta.optimizeWrite.binSize" -> "2048"
  )
  private val sparkEventsTarget = BronzeTargets.sparkEventLogsTarget
  lazy private[overwatch] val executorsModule = Module(2003, "Silver_SPARK_Executors", this, Array(1006))
    .withSparkOverrides(sparkExecutorsSparkOverrides)

Reply:
I could see an advantage to this suggestion if that `Map` (or the `->` tuple inside) were used in multiple statements, but adding an additional reference just increases the reader's cognitive load, IMHO.  Let's revisit this when we identify other modules where this config element will improve their performance as well.  Thanks.